cypress/integration/brewery/ScenarioListOrder.spec.js

```diff
@@ -0,0 +1,59 @@
+import { Login, ScenarioManager, Scenarios } from '../../commons/actions';
+import utils from '../../commons/TestUtils';
+import { DATASET, RUN_TEMPLATE } from '../../commons/constants/brewery/TestConstants';
+
+describe('check scenario sorted list after reload', () => {
+  const scenarios = Array(3)
+    .fill('A - ' + utils.randomStr(7), 0)
```

Comment:
Could we add a " - Test cypress" suffix to the scenario name ? (we can't use a prefix for this test like in other tests ^^)